src/management-system-v2/components/process-list.tsx

```diff
@@ -117,9 +124,33 @@ const ProcessList: FC<ProcessListProps> = ({
             }}
           />
         </Tooltip>
-        <Tooltip placement="top" title={'Delete'}>
-          <DeleteOutlined />
-        </Tooltip>
+        <ProcessEditButton
```

Comment:
Missing Auth check, in this case it should be done with `<AuthCan resourse={toCaslResource('Process', object)} action ...`